Mathlib/Algebra/Polynomial/Div.lean

```diff
@@ -251,6 +264,84 @@ theorem divByMonic_eq_zero_iff [Nontrivial R] (hq : Monic q) : p /ₘ q = 0 ↔
     have : ¬degree q ≤ degree p := not_le_of_gt h
     unfold divByMonic divModByMonicAux; dsimp; rw [dif_pos hq, if_neg (mt And.left this)]⟩
 
+section mem_pow_natDegree_mul
+
+local notation "°" => Polynomial.natDegree
```

Comment:
Not so excited about these, I've got to say

Reply:
I've replaced the notations by something nicer. I do think we should keep the notations, because they make the proof significantly more readable